docs/sink_connector.rst

```diff
@@ -0,0 +1,139 @@
+JDBC Sink Connector
+===================
+
+The JDBC sink connector allows you to export data from Kafka topics to any relational database with a JDBC driver.
+By using JDBC, this connector can support a wide variety of databases without requiring custom code for each one.
+The connector polls data from Kafka to write to the database, based on the topics subscription.
+It is possible to achieve idempotent writes with upserts.
+Auto-creation of tables, and limited auto-evolution is also supported.
+
+Quickstart
+----------
+
+To see the basic functionality of the connector, we'll be copying from a single topic to a local SQLite database.
+This examples assume you are running Kafka and Schema Registry locally on the default ports.
+
+.. note::
+    We use SQLite in these examples, but you can use your favorite database.
+    Follow the same steps, but adjust the ``connection.url`` setting for your database.
+    Confluent Platform includes JDBC drivers for SQLite and PostgreSQL,
+    but if you're using a different database you'll also need to make sure the JDBC driver is available on the Kafka Connect process's ``CLASSPATH``.
+
+Let's start up the connector.
+
+.. sourcecode:: bash
+
+    $ ./bin/connect-standalone etc/schema-registry/connect-avro-standalone.properties etc/kafka-connect-jdbc/sink-quickstart-sqlite.properties
+
+Now, we will produce a record into the `orders` topic.
+
+.. sourcecode:: bash
+
+    $ bin/kafka-avro-console-producer \
+     --broker-list localhost:9092 --topic orders \
+     --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"id","type":"int"},{"name":"product", "type": "string"}, {"name":"quantity", "type": "int"}, {"name":"price",
+     "type": "float"}]}'
+
+The console producer is waiting for input. Copy and paste the following record into the terminal:
+
+.. sourcecode:: bash
+
+    {"id": 999, "product": "foo", "quantity": 100, "price": 50}
+
+Now if we query the database, we will see that the `orders` table was automatically created and contains the record.
+
+.. sourcecode:: bash
+
+    $ sqlite3 test.db
+    sqlite> select * from orders;
+    foo|50.0|100|999
+
+Features and Limitations
+------------------------
+
+Data mapping
+^^^^^^^^^^^^
+
+The sink connector requires knowledge of record schemas, so you should use a suitable converter e.g. the Avro converter that comes with the schema registry,
+or the JSON converter with schemas enabled.
+Record keys if present can be primitive types or a struct, and the record value must be a struct.
```

Comment:
Hmm, some of this will be confusing to Connect newbies since they may not know the difference between Record vs struct. Maybe use "Kafka record" to clarify?